db/fixtures/server_roles.csv

```diff
@@ -1,5 +1,6 @@
 name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
+cockpit_ws,Cockpit,1,false,region
```

Comment:
@petervo do we anticipate having a single cockpit_ws worker in the region talking the cockpit protocol all of the cockpit servers in the region?  What happens if users segment their region into network zones with very limited number of ports reachable between zones?

If cockpit_ws is running in the ui zone as I imagine they would be collocated with the UI workers handing user requests, this zone often doesn't have access to other network locations so I don't know if we'd be able to communicate via the cockpit protocol.

Is it only ssh that needs to be enabled on the firewall?